src/vs/editor/contrib/codeAction/browser/codeActionMenu.ts

```diff
@@ -97,8 +113,9 @@ const TEMPLATE_ID = 'codeActionWidget';
 const codeActionLineHeight = 24;
 const headerLineHeight = 26;
 
-class CodeMenuRenderer implements IListRenderer<ICodeActionMenuItem, ICodeActionMenuTemplateData> {
+let showDisabled = false;
```

Comment:
This is ok for now, but can you add a comment that this should likely be stored in user storage?  That way it will be preserved across windows and on reload

You can also take a look at `IStorageService` if you want to try fixing this before Friday